### PR TITLE
Added Updating Feature

### PR DESCRIPTION
**Pull request by KurzaCationer**
Make sure to change your config, It'll probably crash otherwise

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,7 @@ config.toml
 # Node
 node_modules/
 package-lock.json
+
+# Windows Dev
+start.bat
```
Comment (codecat) [inline .gitignore:9]: I don't see an actual `start.bat` in the final PR

**Comment by KurzaCationer** [comment]
Uhm, it's the .gitignore so it's not supposed to be there

**Comment by codecat** [comment]
...right, that makes a lot of sense. My bad. I'm tired. What's that file supposed to do anyway? Couldn't you use npm scripts/commands for this?

**Comment by KurzaCationer** [comment]
It gets the latest version straight from github with an inchat command.

**Pull request closed without merging**